CanHazFunny/CanHazFunny/IJokeOutput.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System;
+
+public interface IJokeOutput
+{
+	public void TellJoke(string joke);
```

Comment:
```suggestion
	void TellJoke(string joke);
```
Don't use access modifiers in your interface